src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/WebView/WebView_Title.xaml.cs

```diff
@@ -11,5 +12,17 @@ public WebView_Title()
 		{
 			this.InitializeComponent();
 		}
+
+		public void OnGoClicked(object sender, RoutedEventArgs e)
+		{
+			if (Uri.TryCreate(UriInput.Text, UriKind.Absolute, out var uri))
```

Comment:
why not use the (old) `Uri` constructor ? as it will throw an `ArgumentException` if the URI string is invalid

See https://learn.microsoft.com/en-us/dotnet/api/system.uri.-ctor?view=net-9.0#system-uri-ctor(system-string-system-urikind)